PWGHF/HFC/Tasks/taskFlow.cxx

```diff
@@ -563,16 +1240,25 @@ struct HfTaskFlow {
       return size;
     };
 
+    /*
+    auto getTracksSize = [&candidates, this](FilteredCollisionsWSelMult::iterator const& col) {
+      auto associatedTracks = candidates.sliceByCached(o2::aod::track::collisionId, col.globalIndex(), this->cache);
+      auto size = associatedTracks.size();
+      return size;
+    };
+    */
```

Comment:
Better to use `//` on each line. It's better for parsing per line with code analysis tools.

Reply:
Replaced all /**/ with //